### ISSUE TITLE
hasura 2.0.3 and dipdup 2.0.6 don't work

### ISSUE THREAD

**Issue by bb4L** [dipdup-net/dipdup-py#120]
dipdup is trying to query `public_dipdup_state`

as far as i could investingate the correct query might be `dipdup_state`, if i'm not mistaking... 

```
File "/usr/local/lib/python3.9/site-packages/dipdup/hasura.py", line 309, in _get_fields raise HasuraError(f'Unknown table `{name}`') from edipdup.hasura.HasuraError: Unknown table `public_dipdup_state`
```

relevant line(s) of code:

https://github.com/dipdup-net/dipdup-py/blob/7cf40a825db6b6fbcd3211a7b2414b9e683ae1b7/src/dipdup/hasura.py#L281

( using hasura 1.3.3 with dipdup 2.0.5 works )

**Comment by droserasprout** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Hasura error when schema_name is undefined](https://trello.com/c/bAGzElL7/220-hasura-error-when-schemaname-is-undefined)

**Comment by droserasprout** [comment]
@bb4L, @sbihel, fixed in 2.0.7, Docker images will be available soon.

A small feature (probably no one knew about it except me) was removed in this release. Before today, DipDup tried to merge existing Hasura metadata: tables, queries and REST endpoints. This allowed to use a single Hasura deployment with multiple DipDup instances. Could save some time developing multiple backends at once. However, the following problems revealed:

* It's hard to get underlying database schema with Hasura introspection queries
* Orphaned entities must be tracked
* Code complexity grows as edge cases are discovered

So I decided to remove corresponding code and reset Hasura metadata on every configuration. Please reply in this thread or Baking Bad chats if you'd like to have an ability to inject arbitrary metadata (in addition to queries from `project/graphql` directory) on Hasura configuration.

**Comment by bb4L** [comment]
@droserasprout
i updated to dipdup 2.0.7 and hasura 2.0.3 now i get: ```Unknown table `hat` ```

am i missing something? / some configurations?

**Comment by droserasprout** [comment]
@bb4L, please post here or PM me code of `models.py` module of your project, so I can reproduce this issue.

**Comment by droserasprout** [comment]
Fixed in 2.0.8.
